### DISCUSSION TITLE
GSoC contributors?

### DISCUSSION THREAD

**Discussion by Slackadays** [rui314/mold#965] [Ideas]
I've heard that mold is looking for contributors to help out with features and bug fixing, and that these efforts haven't been very successful. Would it be possible to offer mold up in Google Summer of Code? This would essentially be the nuclear option, but there's great potential to make good progress with lots of contributors available.

**Comment by rui314** [comment]
I'm not eagerly trying to recruit contributors, so I think your assumption isn't correct. Besides that, this is still mostly a single person project, and GSoC isn't probably a good fit for it.
